cves/kernel/CVE-2013-0290.yml

```
@@ -356,8 +377,10 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
+    A flag check was forgotten to ensure zero-length data was not passed into the system, which
+    caused the vulnerability.
```

Consider this slight refactor for 80 char line limit.
```suggestion
  note: |
    A flag check was forgotten to ensure zero-length data was not passed into
    the system, which caused the vulnerability.
```